src/Application.js

```diff
@@ -20,31 +20,33 @@ class Application extends Component {
   }
 
   addGrudge = grudge => {
-    API.graphql(graphqlOperation(createGrudge, grudge)).then(response => {
-      const newGrudge = response.data.createGrudge
-      console.log(response)
-      this.setState({ grudges: [newGrudge, ...this.state.grudges] });
-    })
-  };
+    API.graphql(graphqlOperation(createGrudge, { input: grudge })).then(
```

Comment:
Found out that for a mutation the `graphqlOperation()` function takes in two params, the first is the alias name of the mutation you want to do, the second is an object of the variables that you want to pass into that mutation.  Since the autogen took those four keys and lumped them into a GraphQL input, the only thing that had to be done was to feed in the four key value pairs in the grudge object as the value of the input key - we were so close like ten times I feel but hopefully this will work for you - lmk!

Reply:
Hopefully the rebuild of the amplify structure worked.